addon/templates/components/frost-sort-item.hbs

```diff
@@ -12,17 +12,20 @@
   class=(concat css '-direction')
   hook=(concat hookPrefix '-direction')
   hookQualifiers=hookQualifiers
-  size='medium'
+  size='small'
   priority='tertiary'
   onClick=(action 'changeDirection')
 }}
-  {{frost-icon
-    class=(if _descending 'descending')
-    hook=(concat hookPrefix '-direction-icon')
-    hookQualifiers=hookQualifiers
-    icon='direction'
-    pack='frost-sort'
-  }}
+  <div class='icon'>
```

Comment:
Adding an extra layer of DOM to avoid refactoring the code to be how it should really be is not good practice.  Plus, I know that @cstolli will be particularly bothered by this :-)